app/src/main/java/com/github/mateo762/myapplication/habits/fragments/week/WeekFragment.kt

```diff
@@ -65,4 +125,22 @@ class WeekFragment : Fragment(), WeekView.EventClickListener, WeekView.EventLong
     override fun onEmptyViewLongPress(time: Calendar) {
         // Handle empty view long press
```

Comment:
Is this intentionally left empty?

Reply:
Yes, we don't need to handle any clicks with the calendar